### PR TITLE
Minimal akka management

### PR DESCRIPTION
**Pull request by ignasi35**

### CODE REVIEW

File: docs-source/docs/modules/how-to/pages/enable-TLS.adoc [hunk 1]
```diff
@@ -17,7 +17,7 @@ Follow the {akka-http}/server-side/server-https-support.html[Akka HTTP documenta
 
 Note that since gRPC requires HTTP/2, when you enable TLS for gRPC you also need to support https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation[Application-Layer Protocol Negotiation (ALPN)] to negotiate whether both client and server support HTTP/2. The JVM provides ALPN support starting from JDK 8u252. Make sure you run a JVM greater than that.
 
-NOTE: See also the documentation on {akka-http}/current/server-side/http2.html#use-newserverat-bind-and-https[using the `newServerAt(...).bind()` API when serving HTTPS].
+NOTE: See also the documentation on {akka-http}/server-side/http2.html#use-newserverat-bind-and-https[using the `newServerAt(...).bind()` API when serving HTTPS] {akka-http}/server-side/http2.html#use-newserverat-bind-and-https[{tab-icon}, window="tab"]
```
Comment (ignasi35) [inline docs-source/docs/modules/how-to/pages/enable-TLS.adoc:20]: This change is unrelated to the PR. I just stumbled upon this broken link.

File: docs-source/docs/modules/concepts/nav.adoc [hunk 2]
```diff
@@ -28,7 +28,6 @@
 ** xref:ddd.adoc[Brief Summary of DDD concepts]
 ** xref:internal-and-external-communication.adoc[]
 ** xref:services.adoc[]
-** xref:service-discovery.adoc[]
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/nav.adoc:31]: This is unrelated to the PR. It was just broken.

File: docs-source/docs/modules/concepts/pages/akka-management.adoc [hunk 3]
```diff
@@ -5,10 +5,20 @@
...
-ifdef::todo[TODO: this page is WIP]
+== Management
 
+The main building block in {akka-management}[Akka Management] {akka-management}[{tab-icon}, window="tab"]is the Management HTTP server.
+
+This HTTP server runs on a dedicated port, so you can separate the application traffic from the management traffic. It exposes, by default, some read-only endpoints where you can verify the status of the cluster or the xref:how-to:health-checks.adoc[health] of the node.
 
-== Management
 == Bootstrap
-== Service Discovery
```
Comment (ignasi35) [inline docs-source/docs/modules/concepts/pages/akka-management.adoc:14]: The concept of Service Discovery is explained elsewhere. This section could have included an intro to the Akka Service Discover API but I thought it was unnecessary. We can bring it back, but if we do let's put it on the main akka.adoc page since Service Discovery is in Akka core now.

**Comment by ignasi35** [comment]
@octonato this should be a quick review...

**Review by patriknw — APPROVED** [review]
LGTM
